controllers/courseController.js

```diff
@@ -0,0 +1,50 @@
+const UnitOfWork = require('../data/UnitOfWork');
+
+const uow = new UnitOfWork();
+
+const getAll = (req, res) => {
+  res.send(uow.courseRepo.getAll());
+};
+
+const getByID = (req, res) => {
```

Comment:
You should be consistent with this convention. here your controller method is `getByID` with a capital D in the id and you call the unit of work `getById` with a lower case d. I tend to prefer the lower case, but whatever you decide you need to be consistent throughout the project.